bql/semantic/expression.go

```diff
@@ -129,54 +145,59 @@ func (e *evaluationNode) Evaluate(r table.Row) (bool, error) {
 	case GT:
 		return csEL > csER, nil
 	default:
-		return false, fmt.Errorf("boolean evaluation require a boolen operation; found %q instead", e.op)
+		return false, fmt.Errorf("boolean evaluation requires a boolean operation; found %q instead", e.op)
```

Comment:
Same situation: `e.op` is an enum, and will not be "pretty printed". 
Example: https://play.golang.org/p/boTxJB5RRIw

Reply:
Done. Thank you very much for the example.